internal/firmware/export_test.go

```diff
@@ -0,0 +1,34 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2023 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package firmware
+
+// UpdateFwInfoPath provides a hook to allow temporary override of the location
+// of the firmware metadata file
+func UpdateFwInfoPath(path string) func() {
```

Comment:
I know this is a testing/internal function, but it's in the `firmware` package, so I think we can drop the `Fw` from the name. Similar below.

Reply:
I renamed this to "meta" which is currently accepted in the spec for its path.